recipes/keras-resnet/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "keras-resnet" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "8ce27ba782d1b45b127af51208aefdceb2de8d2c54646bac5fc786506ce558c0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+    - pytest
+    - keras >=2.2.4
+
+test:
+  imports:
+    - keras
+
+about:
+  home: https://github.com/broadinstitute/keras-resnet
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Keras package for deep residual networks'
+  description: |
+    Keras-ResNet is the Keras package for deep residual networks. It's fast and flexible.
+
+  doc_url: https://github.com/broadinstitute/keras-resnet/blob/master/README.rst
+  dev_url: https://github.com/broadinstitute/keras-resnet
+
+extra:
+  recipe-maintainers:
+    - '0x00b1'
```

This part here needs to be at least you, if you want to add the developers it is fine as well
But you can maintain this recipe alone if you want to
```suggestion
    - '0x00b1'
    - henrykironde
```